docs/decisions/0006-api-specification-for-enterprise-mfes.rst

```diff
@@ -0,0 +1,308 @@
+0006 API Specification for Enterprise Micro-frontends (MFEs)
+************************************************************
+
+Status
+======
+
+Accepted (March 2023)
+
+Context
+=======
+
+This document intends to outline which API endpoints exist or will exist in support of the
+Enterprise MFEs (e.g., frontend-app-learner-portal-enterprise) as well as other consumers,
+or service-to-service communication.
+
+API Endpoints
+=============
+
+GET Retrieve single, redeemable access policy for a course
+------------------------------------------------------
+
+**/api/v1/enterprise-customer/<enterprise_customer_uuid>/course/<course_key>/can_redeem/**
+
+This API endpoint will be called by the enterprise learner portal to understand whether
+the learner is already enrolled in the course (i.e., a prior redemption has been successfully
+fulfilled) and/or which subsidy access policy should be used to redeem the course when a learner
+clicks the "Enroll" button. 
```

Comment:
[note-to-self] The enterprise-subsidy service is really the source of truth about existing redemptions.  A request to this endpoint will end up making a call to that service's `can_redeem` endpoint, which will help answer the question about existing transaction records for the given (learner, content) pair.